apps/tests/aws-runtime/test/test-service.ts

```diff
@@ -806,7 +806,15 @@ const noise = task(
         await check.set({ id, n });
       } catch (err) {
         console.error(err);
-        if (!(err instanceof TransactionConflictException)) {
+        if (
+          !(
+            err instanceof TransactionConflictException ||
+            (!!err &&
+              typeof err === "object" &&
+              "name" in err &&
+              err.name === "TransactionConflictException")
+          )
```

Comment:
This is not good ...

Do we have a better solution to this?

Reply:
I can't remember why I was catching an AWS error here...